.rubocop.yml

```diff
@@ -0,0 +1,22 @@
+AllCops:
+  TargetRubyVersion: 2.6
+  NewCops: enable
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: double_quotes
```

Comment:
Not a fan of the simple_quotes ? :p

Reply:
It's the default Rubocop config I believe.